config/environments/test.rb

```diff
@@ -1,6 +1,11 @@
 # frozen_string_literal: true
 
 Rails.application.configure do
+  Capybara::Webkit.configure do |config|
+    config.allow_url('https://s3.eu-central-1.amazonaws.com/cotton-cookbook/missing.jpg')
```

Comment:
This is sort of of credential which should be configurable (or maybe it already is and we can build this url with existing ENVs?)